src/App.js

```diff
@@ -1,289 +1,182 @@
-import React, { useState, useEffect } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import * as React from "react";
```

Comment:
Try to get into the habit of importing React this way, when needed.

Reply:
My laziness is different, I’d rather write `useState("")` instead of `React.useState("")` etc. 
It’s nice that there are options!